crates/chainql-core/src/log.rs

```diff
@@ -0,0 +1,15 @@
+#![allow(unused_imports)]
+
+#[cfg(all(feature = "log", feature = "log-tracing"))]
+compile_error!("Features `log-tracing` and `log` cannot be enabled together");
+
+#[cfg(all(not(feature = "log"), not(feature = "log-tracing")))]
```

Comment:
Both log and tracing have forwarding facades, so both are interchangable if someone wants to use library other than we are.

Given that tracing became de-facto standard and just much better due to opentelemetry compatibility (Spans, log attributes, e.t.c), I think we should drop `log` support, and leave only tracing here.